docs/wiki/feature/goggles.md

```diff
@@ -21,7 +21,7 @@ Adds color tint to sunglasses and other eyewear. Causes raindrops to appear on t
 ## 2. Usage
 
 ### 2.1 Cleaning your goggles
-- To clean your goggles press <kbd>SHIFT</kbd> + <kbd>ALT</kbd> + <kbd>T</kbd>(ACE3 deault key bind `Wipe goggles`)
+- To clean your goggles press <kbd>Ctrl</kbd>+<kbd>⇧&nbsp;Shift</kbd>+<kbd>T</kbd> (ACE3 deault key bind `Wipe goggles`)
```

Comment:
Just noticed there's a typo in the parenthesis at the end here

Reply:
there are generally tons of typos all over the place. Someone should run a typo check tool over everything :D